### ISSUE TITLE
Profiled submodule shows only CPU usage -- no values in Memory column

### ISSUE THREAD

**Issue by alecstein** [plasma-umass/scalene#532]
**Describe the bug**
I have a script `main.py` that imports another module, `utils.py`. 

`main.py`
```
from utils import func1
...
```

I want to profile all of the functions (not just func1) in `utils.py` after calling main. But I don't see any output in the Memory column.

**Expected behavior**
I expect the rows profiled in `utils.py` to show values in the Memory column. I tried `--profile-only 'utils'`, but I still don't see values in that column.

**Screenshots**
<img width="821" alt="image" src="https://user-images.githubusercontent.com/16236421/210240124-8a6bc331-7f24-44fe-9726-be2c0dc43c5f.png">

**Desktop (please complete the following information):**
 - macOS Monterey
 - Firefox 108.0.1
 - Scalene version 1.5.16 (2022.12.08)

If you have not yet tried with the repository version (`python3 -m pip install git+https://github.com/plasma-umass/scalene`), please try that before reporting.

**Comment by emeryberger** [comment]
Unless code increases or decreases memory consumption by at least 10MB, it won't show up in a profile. It might be useful to try to run the code with memray, for example (a memory-only profiler that reports memory usage at peak point of consumption) to see what it shows. If the consumption is more than 10MB, this is a possible bug, and we'd love more info.

PS Though I doubt this makes a difference, you can always try again with the latest release (1.5.17).

**Comment by alecstein** [comment]
@emeryberger If I run `scalene` on a test script which contains all the functions in `utils`, they get profiled correctly. 

<img width="952" alt="image" src="https://user-images.githubusercontent.com/16236421/210257602-d799e44f-c291-48b2-9494-0b17d07833cd.png">

It seems to me that if I import a function `func1` from some module, and that function calls other functions (`subfunc1` and `subfunc2`), then scalene does not report their memory usage. On the other hand this seems like something you would have caught so I'm wondering if I'm just using Scalene incorrectly.

**Comment by alecstein** [comment]
I tried to make a MWE but couldn't repro:

`main.py`
```python
from utils import forloop

forloop()
```
`utils.py`
```python
def forloop():
	vals = []
	for i in range(100_000_000):
		vals.append(i)

def expensive_fun(n):
	return n**50
```

This returns what you'd expect:

<img width="896" alt="image" src="https://user-images.githubusercontent.com/16236421/210268904-e315ae74-bcd3-4076-a96a-9922c2401d66.png">

**Comment by alecstein** [comment]
Let me know if you'd like me to give you the actual script to run.

**Comment by emeryberger** [comment]
Definitely. If you don't want to post it here, you can send it to my e-mail account (emery.berger@gmail.com). Thanks in advance.

**Comment by csg524** [comment]
> @emeryberger If I run `scalene` on a test script which contains all the functions in `utils.py`, they get profiled correctly.
> 
> <img alt="image" width="952" src="https://user-images.githubusercontent.com/16236421/210257602-d799e44f-c291-48b2-9494-0b17d07833cd.png">
> 
> It seems to me that if I import a function `func1` from some module, and that function calls other functions (`subfunc1` and `subfunc2`) within the same module, then scalene does not report the memory usage of `subfunc1` and `subfunc2`, but only the combined usage of `func1` from the script I ran `scalene` on.
> 
> On the other hand this seems like something you would have caught so I'm wondering if I'm just using Scalene incorrectly.

For what it's worth, I also experience this. Am I also using scalene wrong, or is this an issue? Apologies for bumping an old-ish thread.

**Comment by emeryberger** [comment]
Hi @chrisstuartparry - can you try this again on the latest version of Scalene?
